pkgs/development/python-modules/ml-collections/default.nix

```diff
@@ -0,0 +1,51 @@
+{ absl-py
+, buildPythonPackage
+, contextlib2
+, fetchPypi
+, fetchurl
+, lib
+, pyyaml
+}:
+
+let
+  requirements = fetchurl {
+    url = "https://raw.githubusercontent.com/google/ml_collections/7f749a281c69f9d0b339c05ecb94b80d95029f25/requirements.txt";
+    sha256 = "1xb351hiscj4zmajfkql3swpacdp6lmz8iwdvwwdx2zqw9a62zps";
+  };
+  requirements-test = fetchurl {
+    url = "https://raw.githubusercontent.com/google/ml_collections/7f749a281c69f9d0b339c05ecb94b80d95029f25/requirements-test.txt";
```

Comment:
Can this re-use version?

Reply:
Not easily. The releases are not tagged on GitHub 😕

Reply:
I fixed this in https://github.com/google/ml_collections/pull/9

Reply:
I just tried refactoring to fetchpatch https://github.com/google/ml_collections/pull/9 instead of copying in the files manually. Unfortunately, this does no good since the current pypi releases still do not include requirements.txt. I would fetchFromGitHub except that they don't have any of the releases tagged at the moment. See https://github.com/google/ml_collections/issues/8.